index.html

```diff
@@ -4264,6 +4264,73 @@ <h3>Spectrum of Privacy</h3>
+href="https://www.w3.org/TR/UAAG20/#def-user-agent">user agent</a>, to preserve
+that trust by not uploading their personal information to a data broker;
+similarly, entities filling the roles in the ecosystem described by this
+specification trust the software that operates on behalf of each of those roles.
+Examples include the following:
+        </p>
+
+        <ul>
+          <li>
+An <a>issuer's</a>
+<a href="https://www.w3.org/TR/UAAG20/#def-user-agent">user agent</a>
+(issuer software), such as an online education platform, is expected to only
+issue <a>verifiable credentials</a> to individuals that have completed their
+education.
+          </li>
+          <li>
+A <a>verifier's</a>
+<a href="https://www.w3.org/TR/UAAG20/#def-user-agent">user agent</a>
+(verification software), such as a hiring website, is expected to only allow
+access to individuals with a valid verification status for
+<a>verifiable credentials</a> and <a>verifiable presentations</a> provided to
+the platform.
+          </li>
+          <li>
+A <a>holder's</a>
+<a href="https://www.w3.org/TR/UAAG20/#def-user-agent">user agent</a>
+(holder software), such as a digital wallet, is expected to only divulge
+information to a <a>verifier</a> when the <a>holder</a> has consented to
+the release of information.
+          </li>
+        </ul>
+
+        <p>
+The examples above are not exhaustive, and the users in these roles can also
+expect a variety of other things from the software they use to achieve their
+goals. That is, the software is expected to operate in the best interests
+of the user, and a violation of that expectation is a violation of trust
+that will result in the software being replaced by something that does not
+violate that trust. Implementers are strongly advised to write software that
+does not violate the trust of the users it will serve. Implementers are
+also advised to provide auditing features in the software that they create
+such that the users, or trusted third parties, can audit that the software is
+behaving in their best interests.
+        </p>
+
+        <p>
+Readers are advised that some software, such as a website that provides services
+to a single <a>verifier</a> and multiple <a>holders</a>, might operate as a
+<a href="https://www.w3.org/TR/UAAG20/#def-user-agent">user agent</a> to both
+roles, but might not always be able to simultaneously operate in the best
+interests of all parties. For example, if that website detects an attempt at
+fraudulent <a>verifiable credential</a> use among multiple <a>holders</a>, it
+might report such an anomaly to the <a>verifier</a>, which would not be in the
+best interest of the <a>holder</a> committing the violation, but would be in the
+best interest of the <a>verifier</a>. It is strongly advised that when software
+operates in this manner, that it is made clear whose best interest in which the
+software is operating though mechanisms such as a website usage policy.
```

Comment:
```suggestion
might report such an anomaly to the <a>verifier</a>, which might be considered to
not be in the best interest of the <a>holder</a> committing the violation, but
would be in the best interest of the <a>verifier</a> as well as any <a>holders</a>
<em>not</em> committing such a violation. It is strongly advised that mechanisms
such as website usage policies be used to make clear the hierarchy of user classes
being protected, when the software is being used to protect the interests of
multiple classes of user.
```

Reply:
@TallTed I took the following change "might report such an anomaly to the <a>verifier</a>, which might be considered to not be in the best interest of the <a>holder</a> committing the violation, but would be in the best interest of the <a>verifier</a> as well as any <a>holders</a> <em>not</em> committing such a violation." text

... but didn't take: "It is strongly advised that mechanisms such as website usage policies be used to make clear the hierarchy of user classes being protected, when the software is being used to protect the interests of multiple classes of user."

... because it wasn't clear what was meant by "the hierarchy of user classes being protected". I believe the exiting language, as corrected by @brentzundel, is more clear. Let me know if you disagree and/or feel strongly about a particular/other change (and I can apply it to `main` since it would be editorial).

Reply:
A nap between readings helps. That sentence could become this, which I think 

When the software is being used to protect the interests of multiple classes of user, it is strongly advised that mechanisms such as website usage policies be used to make clear the classes of user whose interests are being protected.